dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes in this file appear to be adding a new imageset for a placeholder image called "Placeholder". This is a common practice to have a placeholder image available for cases where the actual image is not yet loaded or available.

The changes are straightforward and follow the expected format for an image asset in an Xcode project. The various scale versions (1x, 2x, 3x) are included, which is necessary for supporting different device resolutions.

Overall, the changes in this file seem appropriate and necessary for the application's functionality.